src/index.css

```diff
@@ -22,6 +23,60 @@ code {
     Menlo,
     Monaco,
     Consolas,
-    'Courier New',
+    "Courier New",
     monospace;
 }
+
+/* common styles */
+main {
+  padding: 0 6em;
+}
+
+button {
+  cursor: pointer;
+}
+
+::-webkit-input-placeholder { /* Edge */
+  color: #e8e8e8;
+}
+
+:-ms-input-placeholder { /* Internet Explorer 10-11 */
+  color: #e8e8e8;
+}
+
+::placeholder {
+  color: #e8e8e8;
+}
```

Comment:
- [ ] Please don't repeat the same rule for multiple classes, a better option would be to do it like this:

```css
::-webkit-input-placeholder, :-ms-input-placeholder, ::placeholder {
  color: #e8e8e8;
}
```
Let us follow CSS best practices and apply these changes wherever needed to comply with the project requirements, please. (This includes color, rules and fonts repetition if any applies)